pontoon/base/templates/user_settings.html

```diff
@@ -0,0 +1,32 @@
+{% extends "landing.html" %}
+
+{% import 'widgets/locales.html' as locales %}
+
+{% block extend_css %}
+  {{ super() }}
+
+  {% stylesheet 'multiple_locale_selector' %}
+{% endblock %}
+
+{% block extend_js %}
+  {{ super() }}
+  {% javascript 'multiple_locale_selector' %}
+{% endblock %}
+
+{% block heading %}Settings{% endblock %}
+{% block title %}Pontoon &middot; User locale settings{% endblock %}
+{% block subtitle %}Set the order of suggestions{% endblock %}
+
+{% block middle %}
+  <form method="POST" action="{{ url('pontoon.user_settings') }}" class="user-locales-settings">
+    {% csrf_token %}
+
+    <div class="clearfix">
+      {{ locales.select(available_locales, selected_locales, form_field='locales_order', sortable=True) }}
+    </div>
+    <div class="controls">
+      <a class="cancel" href="/">Cancel</a>
```

Comment:
Cancel should take us back to the previous page if openned from the translate view or the profile page.